frontend/appflowy_flutter/lib/workspace/presentation/widgets/float_bubble/question_bubble.dart

```diff
@@ -199,12 +208,48 @@ class FlowyVersionDescription extends CustomActionCell {
   }
 }
 
+class FlowyUpdateStatus extends CustomActionCell {
+  @override
+  Widget buildWithContext(BuildContext context) {
+    return BlocProvider.value(
+      value: getIt<VersionCheckerBloc>(),
+      child: BlocBuilder<VersionCheckerBloc, VersionCheckerState>(
```

Comment:
I suggest using a button instead of text, allowing users to click it to open the GitHub release page. Alternatively, for future updates, we could support in-app updates, but I'm not sure if any Flutter packages have implemented this feature.

Reply:
For auto-updates, I was thinking about creating a separate project that would be shipped with the application.

AppFlowy can then launch the application, the application fetches the new archive, closes AppFlowy, replaces files, and then launches AppFlowy again.

I intend to build a small sample with gRPC to test it out in Q2.

Reply:
So, temporarily use a button to make it easier to access the release link?

Reply:
Yeah, I'll add that tomorrow or Monday :+1:

Reply:
Any updates?